Makefile

```diff
@@ -93,17 +93,14 @@ test-browser:
 	cd test && node ../node_modules/webpack/bin/webpack.js unittest_browser.js -o dist/unittest_browser.bundle.js
 	cd test && open unittest.html
 
-# Note: versions < 0.12 are *not* supported.  The 'beforeExit' event has
+# Note: versions < 6.2 are *not* supported.  The 'beforeExit' event has
```

Comment:
Any particular reason we don't support 6.0 or 6.1?

Reply:
Not sure - 6.0/6.1 weren't being tested beforehand.